src/charm.py

```diff
@@ -355,6 +367,10 @@ def _pod_ip(self) -> str:
 
         return bind_address
 
+    @property
+    def _web_route_prefix(self) -> str:
```

Comment:
Is "web route prefix" standard terminology? Should this be name "subpath" or "ingress_subpath" instead?

Reply:
It's how prometheus calls it. I saw no reason to add to the confusion with a different term.